config_defaults.yml

```diff
@@ -50,14 +50,14 @@
     - '/spec/'
 .travis.yml:
   ruby_versions:
-    - 2.5.1
+    - 2.5.3
   global_env:
-    - BEAKER_PUPPET_COLLECTION=puppet6 PUPPET_GEM_VERSION="~> 6.0"
```

Comment:
Dropped the `.0` to make this easier to read. `~> 6` is clearer that it will match the latest in the Puppet 6 series.